### PR TITLE
feat: add badges endpoints description

### PR DESCRIPTION
**Pull request by shmbajaj**
## Meta Info
- Closes (https://github.com/Real-Dev-Squad/website-members/issues/469)
- [PR](https://github.com/Real-Dev-Squad/website-backend/pull/832)

## Before Change
- Badge object type/props
```js
{
  'id': string,
  'title': string,
  'description': string,
  'imgUrl': string,
  'users': [
            rdsUserId,
      ],
}
```
- Info: description, params, query, body, headers, cookie and response available for this endpoint only
```js
GET /badges
```

## After Change
- Changed badge object type/props
```js
{
  'id': string,
  'name': string,
  'description': string,
  'imageUrl': string,
  'createdBy': string,
  'createdAt': { 'date': string, 'time': string }
}
```
- Added description, params, query, body, headers, cookie and response for these endpoints
```js
GET /users/:userId/badges
GET /badges
POST /badges
POST /badges/assign
DELETE /badges/remove
```

### CODE REVIEW

File: badges/README.md [hunk 1]
```diff
@@ -5,20 +5,24 @@
...
 
 ## **Requests**
 
-|              Route               |           Description            |
-|:--------------------------------:|:--------------------------------:|
-|    [GET /badges](#get-badges)    | Returns all badges in the system |
+|              Route                                                      |           Description                   |
+|:-----------------------------------------------------------------------:|:---------------------------------------:|
+|    [GET /badges](#get-badges)                                           |   Returns all badges in the system      |
+|    [GET /badges/:username](#get-badgesusername)                         |   Returns badges assigned to the user   |
```
Comment (ivinayakg) [inline badges/README.md:21]: /users/:id/badges

File: badges/README.md [hunk 2]
```diff
@@ -5,20 +5,24 @@
...
+|    [POST /badges](#post-badges)                                         |   Creates a new badge                   |
+|    [POST /badges/assign/:username](#post-badgesassignusername)          |   Assigns a badge to the user           |
+|    [DELETE /badges/unassign/:username](#delete-badgesunassignusername)  |   Unassigns a badge from user badges    |
```
Comment (ivinayakg) [inline badges/README.md:24]: badges/assign
body : {
user, badges[]
}

File: badges/README.md (hunk 1 above)
Comment (shmbajaj) [reply badges/README.md]: [Description: GET /users/:id/badges](https://github.com/Real-Dev-Squad/website-api-contracts/pull/103/commits/6abd0f2aa92c478410712af6d0027959f890e7b4#diff-02c1e767355fdce425fcde731db362514a82bcd0025e5ba8db66277e8bb3362cR144)

File: badges/README.md (hunk 2 above)
Comment (shmbajaj) [reply badges/README.md]: Description: [Assign and Unassign Badges](7f36e92ed10393af32e1bfc28f6bfe069493cb39)

**Review by ivinayakg — DISMISSED** [review]

**Review by Maheima — DISMISSED** [review]
LGTM!

**Review by Maheima — APPROVED** [review]
LGTM

**Pull request merged**